### ISSUE TITLE
linearize error with redis backend (maybe python3 related)

### ISSUE THREAD

**Issue by bastianh** [andrewgodwin/channels#48]
while continuing the getting startet tutorial I stumbled when I added the linearize decorator:

```
ERROR django.channels Error processing message with consumer cm_base.consumers.ws_message:
Traceback (most recent call last):
  File "/home/vagrant/pyenv/src/channels/channels/worker.py", line 37, in run
    consumer(message)
  File "/home/vagrant/pyenv/src/channels/channels/decorators.py", line 26, in inner
    locked = message.channel_backend.lock_channel(message.reply_channel)
  File "/home/vagrant/pyenv/src/channels/channels/backends/redis_py.py", line 177, in lock_channel
    return bool(self.connection(self.consistent_hash(channel)).setnx(key, "1"))
  File "/home/vagrant/pyenv/src/channels/channels/backends/redis_py.py", line 41, in consistent_hash
    bigval = binascii.crc32(value) & 0xffffffff
TypeError: 'Channel' does not support the buffer interface
```

**Comment by andrewgodwin** [comment]
This is likely a Python 3 error; I'll look into it in a bit. Might be a side-effect of the fix to #24.

**Comment by andrewgodwin** [comment]
Linearize is gone.
